TEST-PLAN.md

```diff
@@ -0,0 +1,109 @@
+# DP Creator II Test Plan
+
+## Introduction
+
+### Purpose
+
+Automate the end-to-end testing of the application to ensure it meets user requirements and maintains reliability, especially in the context of privacy-preserving data processing.
+
+### Project Overview
+
+The single user application will be installed with pip, and offer a web application (running on localhost) that guides the user through the application of differential privacy on a data file they provide.
+
+## Scope
+
+### In-Scope
+
+- Starting the application with and without `--demo` and other CLI flags.
+- Form filling and navigation between tabs.
+- Conditionally disabled controls (for example, controls late in the flow when inputs have not been provided, and controls early in the flow after the release has been made).
+- Report export.
+- Runnability of any generated code.
+
+### Out-of-Scope
+
+- Performance testing: Test timeouts may incidentally identify components which are slow ([issue](https://github.com/opendp/dp-creator-ii/issues/116)), but it's not the focus of automated testing.
```

Comment:
```suggestion
- Performance testing: Test timeouts may incidentally identify components which are slow ([issue](https://github.com/opendp/dp-creator-ii/issues/116)), but performance is not the focus of automated testing.
```

With "it's" I get slightly confused. I don't think you're referring to "test timeouts", which would be "they're".